cli-plugins/apps/getting-started/configuration.hbs.md

```diff
@@ -0,0 +1,168 @@
+
+However, sometimes color, emojis and other characters are not needed (e.g. scripting) or even well 
+interpreted in certain terminals and the best way to suppress them is using the `--no-color` flag.
+
+So, for example, in a workload that is created through local path, which usually shows emojis and a
+progress bar, these special characters would be avoid by using `--no-color`.
+
+```bash
+tanzu apps workload apply my-workload --local-path path/to/my/source -s my-registry.ext/my-project/my-workload --type web --no-color
+The files and/or directories listed in the .tanzuignore file are being excluded from the uploaded source code.
+Publishing source in "path/to/my/source" to "my-registry.ext/my-project/my-workload"...
+Published source
+
+Create workload:
+      1 + |---
+      2 + |apiVersion: carto.run/v1alpha1
+      3 + |kind: Workload
+      4 + |metadata:
+      5 + |  labels:
+      6 + |    apps.tanzu.vmware.com/workload-type: web
+      7 + |  name: my-workload
+      8 + |  namespace: default
+      9 + |spec:
+     10 + |  source:
+     11 + |    image: my-registry.ext/my-project/my-workload:latest@sha256:724bcd14c3a84fc7a918cd8ee7a6a987de1699617a17c5af166e8c689a2becf7
+? Do you want to create this workload? [yN]:
+```
+
+To avoid having color in general for Tanzu Apps Plug-in, the best way is setting the `NO_COLOR`
+environment variable. This will suppress color, emojis and progress bar for all the Plug-in related commands.
+
+```bash
+export NO_COLOR=true
+```
+
+## .tanzuignore file
+
+There are many files and directories in projects that are not connected to running code
+(these files are not part of the final running container). When creating a workload from local
+source code, list these unused files and directories in the `.tanzuignore` file to avoid unnecessary
+consumption of resources when uploading the source.
+
+When iterating on code with the `--live-update` flag enabled, changes to directories or files
+listed in `.tanzuignore` do not trigger the automatic re-deployment of the source code.
+
+The following are some guidelines for the `.tanzuignore` file:
+
+- The `.tanzuignore` file should include a reference to itself, as it provides no value when deployed.
+- Directories must not end with the system separator `/`, or `\`.
+- Comments using hashtag `#` can be included.
+- If the `.tanzuignore` file contains files or directories that are not found in the source code, they are ignored.
+
+### Example of a .tanzuignore file
+
+```console
+.tanzuignore # must contain itself in order to be ignored
+# This is a comment
+this/is/a/folder/to/exclude
+
+this-is-a-file.ext
+```
+
+## Registry flags and environment variables
+
+The Apps CLI plug-in allows users to push images to their private registry by setting some flags 
+when creating workloads from local source code.
+
+A user can either trust a custom certificate on their system or pass the path to the certificate via flags.
```

Comment:
A user can either trust a custom certificate on their system or pass the path to the certificate via flags (environment variables can also be set to avoid having to pass the flags/values for every command incantation) .

Below is a list of each of the flags with their environment variable equivalent in parenthesis:

- `--registry-ca-cert`: This is the path of the self-signed certificate needed for the custom or 
  private registry (`TANZU_APPS_REGISTRY_CA_CERT`).
- `--registry-password`: Use this when the registry requires credentials to push (`TANZU_APPS_REGISTRY_PASSWORD`).
- `--registry-username`: Use with `--registry-password` to set the registry credentials (`TANZU_APPS_REGISTRY_USERNAME`).
- `--registry-token`: Set when the registry authentication is done through a token (`TANZU_APPS_REGISTRY_TOKEN`).

Reply:
the above suggestion is for lines 108-119

Reply:
changed